code/modules/client/preferences_mysql.dm

```diff
@@ -57,7 +56,7 @@
 					UI_style='[UI_style]',
 					UI_style_color='[UI_style_color]',
 					UI_style_alpha='[UI_style_alpha]',
-					be_special='[be_special]',
+					be_role='[sql_sanitize_text(list2params(be_special))]',
```

Comment:
That's why you should always be careful with how Topic() code is written, but I don't think it would be a problem in this specific case.